content/docs/1.10/deployment.md

```diff
@@ -133,7 +133,7 @@ Port  | Protocol | Function
 
 Collectors require a persistent storage backend. Cassandra and Elasticsearch are the primary supported storage backends. Additional backends are [discussed here](https://github.com/jaegertracing/jaeger/issues/638).
 
-The storage type can be passed via `SPAN_STORAGE_TYPE` environment variable. Valid values are `cassandra`, `elasticsearch`, `kafka` and `memory` (only for all-in-one binary).
+The storage type can be passed via `SPAN_STORAGE_TYPE` environment variable. Valid values are `cassandra`, `elasticsearch`, `kafka` (only as a sink) and `memory` (only for all-in-one binary).
```

Comment:
I'm not sure it's immediately clear what a `sink` means here. "Cannot power jaeger-query" is more explicit that kafka cannot be used (solo).

Reply:
I would also prefer a different name than sink, maybe a `buffer`?